retrospective-update/prepare.py

```
@@ -0,0 +1,77 @@
+import subprocess
+import traceback
+from glob import glob
+
+import pandas as pd
+import xarray as xr
+from natsort import natsorted
+
+from cloud_logger import CloudLog
+from set_env_variables import *
```

Ruff (0.12.2)

`from set_env_variables import *` used; unable to detect undefined names

(F403)